index.html

```diff
@@ -27,13 +34,19 @@ <h1>POKEYOU</h1>
 <br>
     <!-- Screen 2 HTML -->
 
-    <label for="avatar">Choose a profile picture:</label>
+    <div id="User Image Container">
+    <label for="userImage">Choose a profile picture:</label>
 
-    <input type="file" id="avatar" name="avatar" accept="image/png, image/jpeg">
+    <input type="file" id="user-image" name="image" accept="image/png, image/jpeg">
+    </div>
 
-    <!-- Adding a submit button -->
+    <br>
```

Comment:
can we make this spacing with CSS?

Reply:
<br> was merely for creating space between the input element and the image container div that follows.